.github/ISSUE_TEMPLATE/weekly-release.md

```diff
@@ -43,14 +51,14 @@ version of the docs. The `-b` flag also bumps the version in the code to match.
 
 ```
 git tag -a v0.x.y -m "Release v0.x.y"
-git push upstream refs/tags/v0.x.y
+git push origin refs/tags/v0.x.y
```

Comment:
I think we need to leave it at `upstream`. Is your setting similar to below?

```
➜  sonobuoy git:(main) git remote -v
origin	git@github.com:YOUR_GITHUB_ID/sonobuoy.git (fetch)
origin	git@github.com:YOUR_GITHUB_ID/sonobuoy.git (push)
upstream	git@github.com:vmware-tanzu/sonobuoy.git (fetch)
upstream	no_push (push)
```

Reply:
I believe `origin` is the default when doing a git clone and that's how mine shows up. https://git-scm.com/docs/git-clone

```
➜  sonobuoy git:(main) git remote -v
origin	git@github.com:vmware-tanzu/sonobuoy.git (fetch)
origin	git@github.com:vmware-tanzu/sonobuoy.git (push)
➜  sonobuoy git:(main)
```

Reply:
I understand where you are coming from. 

The convention in OSS community (at least in k8s community) is to fork the repo, set upstream for the OSS repo and set origin as your forked repo. That is why the instruction says to push to `upstream`.

Reply:
See [k8s github-workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md?plain=1#L45-L62) for more details

Reply:
Ah, I see! I've swapped it back.